core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -30,29 +30,31 @@ oppia.directive('audioTranslationBar', [
 
         $('.oppia-translation-tab').on('dragover', function(evt) {
           evt.preventDefault();
-          if (!scope.showDropArea) {
-            scope.showDropArea = true;
-            scope.$digest();
-          }
+          /* Don't show drag and drop area for user that is logged in
+            and cannot add translation. */
+          scope.dropAreaIsAccessible = !(!scope.userIsGuest &&
```

Comment:
Is this correct? Shouldn't it be `(!scope.userIsGuest && !scope.translationIsDisallowed)`?

Otherwise what you have is equivalent to "scope.userIsGuest || !scope.translationIsDisallowed" which seems like ... not what you want.

Reply:
yeah it's possible to do it that way, so I'm going to do it for simplicity's sake.